addon/components/validation-error-field.js

```diff
@@ -6,12 +6,18 @@ var ValidationErrorField = Ember.Component.extend({
     attributeBindings: ["array", "className", "model", "field", "validation", "submitted", "delayed", "index"],
     initialize: Ember.on("init", function() {
         var field = this.get("field");
-        Ember.Binding.from("model." + field).to("fieldName").connect(this);
-        Ember.Binding.from("model." + field + "IsPrimed").to("isPrimed").connect(this);
-
         var computedKeys = ["validation", "isPrimed", "submitted", "fieldName"];
+
         if(this.get("array")) {
-            computedKeys.push("array.@each." + field);
+            var fields = field.split(",");
+            fields.forEach(function(f) {
+                computedKeys.push("array.@each." + f);
```

Comment:
@cklimkowsky as you mentioned below, "array" is a new feature as of yesterday that allows you to break the cache of all validation components when used in the array context.

for this feature array will be required as we need to re-compute each field (the radio button *and* the checkbox both on any change)